src/main/docker/Dockerfile

```diff
@@ -26,20 +26,16 @@ RUN zypper -n refresh && \
     zypper -n install curl postgresql dejavu-fonts && \
     zypper -n clean --all
 
-# Install gosu
-RUN gpg --batch --keyserver-options http-proxy=${env.HTTP_PROXY} --keyserver hkps://keys.openpgp.org \
-      --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 && \
-    curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-amd64" && \
-    curl -o /usr/local/bin/gosu.asc -SL "https://github.com/tianon/gosu/releases/download/1.12/gosu-amd64.asc" && \
-    gpg --batch --verify /usr/local/bin/gosu.asc /usr/local/bin/gosu && \
-    rm /usr/local/bin/gosu.asc && \
-    chmod +x /usr/local/bin/gosu
+RUN chmod a+rwx /etc/pki/trust/anchors && \
+    chmod -R a+rwx /var/lib/ca-certificates && \
+    chmod -R a+rw /etc/ssl
 
 # Add scripts to be executed during startup
 COPY startup /startup
-ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.19.0/container-cert-script/install-ca-cert.sh \
+ADD https://raw.githubusercontent.com/CAFapi/caf-common/SCMOD-12755/container-cert-script/install-ca-cert.sh \
     /startup/startup.d/install-ca-cert-base.sh
-RUN chmod +x /startup/* /startup/startup.d/*
+RUN chmod +rx /startup/* /startup/startup.d/* && \
+    mkdir -p /tmp/ssl && chmod a+rwx /tmp/ssl
```

Comment:
What is the `/tmp/ssl` directory about?  Is it used by the new `install-ca-cert.sh` script?